state/execution.go

```diff
@@ -198,15 +198,15 @@ func execBlockOnProxyApp(logger log.Logger, proxyAppConn proxy.AppConnConsensus,
 		return nil, err
 	}
 
-	// Run txs of block
+	// Run txs of block.
 	for _, tx := range block.Txs {
 		proxyAppConn.DeliverTxAsync(tx)
 		if err := proxyAppConn.Error(); err != nil {
 			return nil, err
 		}
 	}
 
-	// End block
+	// End block.
```

Comment:
what's up with all these dots at the end? does it really matter?

Reply:
I'd say "End block." is a complete imperative sentence in Tendermint slang where "the" can be omitted before common type nouns like "block".  :P

"Take that".